### ISSUE TITLE
Blank pages when setting limitPagination/how to hide pages?

### ISSUE THREAD

**Issue by Smoz** [alfrcr/paginathing#5]
@alfrcr 

I'm having an issue where if I set the limitPagination param then it's displaying that amount of pages regardless if there is that much information available but if there is more information then those pages generate normally.

I'm building a manga website and I'd like to use pagination on the chapter selection since some manga have a great number of chapters. This information is pulled from a mysql database so each manga can have a different number of chapters which is dynamically generated. Thus I'd like only to show a max of 10 pages at once and not generate more pages than necessary.

This is what I have.
```
<!-- Your script -->
    <script type="text/javascript">
        jQuery(document).ready(function($){
            $('#chapter-list').paginathing({
                perPage: 10, // show item per page
                limitPagination: 10,
                containerClass: 'pagination-container',
                insertAfter: '#chapters'
            })
        });
    </script>
```
This is the output. As you can see after page 1 the rest are blank. 

**Page 1**
![image](https://cloud.githubusercontent.com/assets/25465045/26391898/2e735b46-402a-11e7-84af-d5f4bd0490c1.png)

**Page 2**
![image](https://cloud.githubusercontent.com/assets/25465045/26391907/3910ac0c-402a-11e7-83b9-dc423d59f6d6.png)

However if you set `limitPagination: false` then it appears like this which is expected.

![image](https://cloud.githubusercontent.com/assets/25465045/26391812/d71f0b38-4029-11e7-8426-1c98a547bb2d.png)

Basically I'm trying to avoid something like this.

![image](https://cloud.githubusercontent.com/assets/25465045/26392115/1cbb416a-402b-11e7-8ffa-09e007c942a7.png)

So I want to hide the pages after a certain number like 10. 

![image](https://cloud.githubusercontent.com/assets/25465045/26392136/2e7dca76-402b-11e7-8d82-6ca0b154d582.png)

Do you think you can help me figure out how to hide the pages after a certain number until clicked?

**Comment by alfrcr** [comment]
Thank you for reporting this issue.
I'm wondering about the 2nd page is being blank. It'd good if you have a repo or fiddle that reproduce this issue.
I'll work on it on Monday.

**Comment by Smoz** [comment]
Sorry for the delay! I didn't realize you had responded since I had some computer issues >.<

I can't get JSFiddle to work at all so here are some clips of the issue. Hope it's clear enough.

This first one is with dynamic content and it's set to show 10 perPage and limitPagination is false -https://youtu.be/LxtxGrGzCh0

This second one is with dynamic content and it's set to show 10 perPage and limitPagination is 10 - https://youtu.be/QFGsnfn2cVM

This third one is with static content and it's set to show 10 perPage and limitPagination is 10 - https://youtu.be/fuvTtZbM8-c

**Comment by ilmari-lauhakangas** [comment]
I solved it like this,

old block:
```
    this.totalPages = Math.max(
      Math.ceil(this.totalItems / this.options.perPage),
      options.limitPagination
    );
```
changed to:
```
    this.pageDivision = 0;
    this.pageDivision = Math.ceil(this.totalItems / this.options.perPage);
    if(this.options.limitPagination >= this.pageDivision) {
        this.options.limitPagination = this.pageDivision;
    }
    this.totalPages = Math.max(Math.ceil(this.pageDivision, this.options.limitPagination));
```
I also incorporated the syntax corrections from https://github.com/alfrcr/paginathing/pull/6/

**Comment by diclonius** [comment]
Great, I had the same problem.
Thanks for the input.
